packages/fxa-auth-server/scripts/cleanup-old-carts.ts

```diff
@@ -11,14 +11,28 @@ const pckg = require('../package.json');
 
 const anonymizeableFields = new Set(['taxAddress'] as const);
 
-const parseDeleteBefore = (deleteBefore: string | number) => {
+const parseDeleteBefore = (deleteBefore: string | number | null) => {
```

Comment:
[non-blocking] I believe this should stay as `string | number` since I don't believe this will ever receive `null`.